pallets/exchange/src/tests.rs

```diff
@@ -2509,13 +2470,13 @@ fn simple_sell_sell_with_error_should_not_pass() {
 				IntentionType::SELL,
 				user_2_sell_intention_id,
 				DispatchError::Module {
-					index: 0,
+					index: 2,
```

Comment:
Why did the index change?

Reply:
now, all tests actually use construct_runtime macro. That means  it correctly assigns pallet/module index to each pallet used in tests. Before all module have 0 as module index.